pybaseball/playerid_lookup.py

```diff
@@ -52,7 +52,22 @@ def get_lookup_table(save=False):
     table['name_first'] = table['name_first'].str.lower()
     return table
 
+def name_similarity(last: str, first: str, player_table: pd.DataFrame) -> pd.DataFrame:
+    """Calculates similarity of first and last name provided with all players in player_table
 
+    Args:
+        last (str): Provided last name
+        first (str): Provided first name
+        player_table (pd.DataFrame): Chadwich player table including names
+
+    Returns:
+        pd.DataFrame: player table with levenshtein distance added
+    """
+    player_table["lastname_dist"] = player_table.dropna().apply(lambda row : levenshtein(row["name_last"], last),axis=1)
+    player_table["firstname_dist"] = player_table.dropna().apply(lambda row : levenshtein(row["name_first"],first),axis=1)
+    player_table["total_dist"] = player_table["lastname_dist"] + player_table["firstname_dist"]    
```

Comment:
I don't know if adding them is the best here. maybe use a harmonic mean instead or something?

Reply:
switched to fuzzywuzzy process; no longer need to perform this operation